Src/CZICmd/cmdlineoptions.cpp

```diff
@@ -733,9 +762,17 @@ CCmdLineOptions::ParseResult CCmdLineOptions::Parse(int argc, char** argv)
         ->option_text("PIXELTYPE")
         ->check(generatorpixeltype_validator);
     cli_app.add_option("--cachesize", argument_subblock_cachesize,
-        "Only used for 'PlaneScan' ...")
+        "Only used for 'PlaneScan' - specify the size of the subblock-cache in bytes. The argument is to "
```

Comment:
Just for clarity I think it would be a good idea to add a format example like in tilesize-for-plane-scan